e2e/src/step-definitions/create-new-wallet.steps.ts

```diff
@@ -21,7 +22,7 @@ Given(/I verify my mnemonic/, async () => {
     return Number(numberText);
   });
 
-  const wordInputs = await Pages.VerifyMnemonic.getWordsInputs();
+  const wordInputs = await findElements(NewSeedVerifyTestIds.firstMnemonicInput);
```

Comment:
Due to logic with `Pages.ImportExistingWallet.enterSeedPhrase` and `Pages.ImportAccountMnemonic.enterSeedPhrase` all this code must be moved into some `Pages.VerifyMnemonic.enterSeedPhraseVerification`

Reply:
done